crates/ruff_linter/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -807,10 +808,39 @@ fn check_fixture_returns(checker: &Checker, name: &str, body: &[Stmt], returns:
 }
 
 /// PT019
-fn check_test_function_args(checker: &Checker, parameters: &Parameters) {
+fn check_test_function_args(checker: &Checker, parameters: &Parameters, decorators: &[Decorator]) {
+    let mut named_parametrize = FxHashSet::default();
+    for decorator in get_parametrize_decorators(decorators) {
+        let Some(call_expr) = decorator.expression.as_call_expr() else {
+            continue;
+        };
+        let first_arg = call_expr
+            .arguments
+            .arguments_source_order()
+            .next()
+            .and_then(|arg| {
+                if let ArgOrKeyword::Arg(expr) = arg {
+                    expr.as_string_literal_expr()
+                } else {
+                    None
+                }
+            });
```

Comment:
I think you could use something like this here:

```suggestion
        let first_arg = call_expr
            .arguments
            .find_argument_value("argnames", 0)
            .and_then(Expr::as_string_literal_expr);
```

However, based on the [docs](https://docs.pytest.org/en/stable/reference/reference.html#pytest.Metafunc.parametrize), I'm not sure the check for a string literal is entirely correct:

> Parameters:
argnames ([str](https://docs.python.org/3/library/stdtypes.html#str) | [Sequence](https://docs.python.org/3/library/typing.html#typing.Sequence)[[str](https://docs.python.org/3/library/stdtypes.html#str)]) – A comma-separated string denoting one or more argument names, or a list/tuple of argument strings.

Based on this, I think it can either be a comma-separated string or a list/tuple of strings. It also works with variables, not just literals, but that might be too rare to worry about much:

```python
import pytest

x = "test_input,expected"

@pytest.mark.parametrize(x, [("3+5", 8), ("2+4", 6), ("6*9", 42)])
def test_eval(test_input, expected):
    assert eval(test_input) == expected
```